source/includes/get-started/query-database.java

```diff
@@ -0,0 +1,39 @@
+import com.mongodb.*;
+import com.mongodb.reactivestreams.client.MongoCollection;
+import org.bson.Document;
+
+import reactor.core.publisher.Mono;
+
+import com.mongodb.reactivestreams.client.MongoClient;
+import com.mongodb.reactivestreams.client.MongoClients;
+import com.mongodb.reactivestreams.client.MongoDatabase;
+
+import static com.mongodb.client.model.Filters.eq;
+
+public class QueryDatabase {
+    public static void main(String[] args) {
+        // Replace the placeholder with your Atlas connection string
+        String uri = "<connection string>";
+
+        // Construct a ServerApi instance using the ServerApi.builder() method
+        ServerApi serverApi = ServerApi.builder()
+                .version(ServerApiVersion.V1)
+                .build();
+
+        MongoClientSettings settings = MongoClientSettings.builder()
+                .applyConnectionString(new ConnectionString(uri))
+                .serverApi(serverApi)
+                .build();
+
+        // Create a new client and connect to the server
+        try (MongoClient mongoClient = MongoClients.create(settings))
+        {
```

Comment:
It looks like the opening brace `{` is on a new line. For consistency with the Java coding style above, could we place the `{` on the same line as the try statement? 

```suggestion
        try (MongoClient mongoClient = MongoClients.create(settings)) {
```